build.sbt

```diff
@@ -0,0 +1,36 @@
+enablePlugins(GitVersioning)
+
+git.baseVersion := "2.14.7-sbt"
+
+organization := "org.scala-sbt.jline"
+
+name := "jline"
+
+homepage := Some(url("https://github.com/sbt/jline2"))
+
+description := "patched jline2 for sbt"
+
+val license = 
+  "https://github.com/sbt/jline2/blob/master/src/main/resources/META-INF/LICENSE"
+licenses := List("Apache-2.0" -> url(license))
+
+scmInfo := Some(ScmInfo(url("https://github.com/sbt/jline2"), "git@github.com:sbt/jline2.git"))
+
+developers := List(
+  Developer("eed3si9n", "Eugene Yokota", "@eed3si9n", url("https://github.com/eed3si9n")),
+  Developer("eatkins", "Ethan Atkins", "@eatkins", url("https://www.ethanatkins.com")),
+)
+
+autoScalaLibrary := false
+
+crossPaths := false
+
+libraryDependencies += "org.fusesource.jansi" % "jansi" % "1.12"
+
+/*
+ * This project uses java 6 doc syntax that isn't parseable by 1.7 which makes it
+ * impossible to generate docs even with sbt 0.13.
+ */
+Compile / doc := file("/dev/null")
```

Comment:
We'd need doc for Sonatype publishing, so I'd say we should build this fork on JDK 8.

Reply:
I think an empty doc is actually ok but I could at least change this from /dev/null to the normal doc directory. The only way to get doc to work would probably be to delete all of the doc tags or figure out how to use the java6 doc tool but jabba doesn't even have a working java 6 anymore so it's kind of a pain.

Reply:
Here's my PR for sbt build with Travis CI running the tests - https://github.com/sbt/jline2/pull/2

Reply:
I was curious about sonatype publishing, so I published into the `com.swoval.jline` org: https://repo1.maven.org/maven2/com/swoval/jline/jline/2.14.7-sbt-8d4caeb87217c3a5f57b6884d28cfffcbe3edc86/. It worked fine with `Compile / doc :=  file("/dev/null")`